package.json

```diff
@@ -50,6 +50,7 @@
 		"semver": "^6.3.0"
 	},
 	"devDependencies": {
+		"@lubien/fixture-beta-package": "^1.0.0-beta.1",
```

Comment:
Is it possible to  mock one  during test?

Reply:
Personally I haven't heard anything of sort. Tho for me this is pretty much all I need since I could handle all known cases so far.